src/unistd.rs

```diff
@@ -756,12 +756,27 @@ pub fn close(fd: RawFd) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// reads from a raw file descriptor.
+/// reads at most as many bytes as will fit into the provided buffer
```

Comment:
Blank line before this one.

Reply:
You put the blank line in the wrong place.  Susurrus wanted the blank to be the second line of the comment block, so the first line would stand alone.

Reply:
This is important because the first line is a brief summary and then there should be a blank line before additional information. I'd suggest reading the [documentation section](https://doc.rust-lang.org/stable/book/first-edition/documentation.html) of the Rust book, since many people don't seem to know the various conventions used by rustdoc.